### DISCUSSION TITLE
Maintenance notification

### DISCUSSION THREAD

**Discussion by decentralizedb** [btcpayserver/btcpayserver#5526] [Ideas / Feature Requests]
I run a server on which several participants have their Stores.
It would be helpful for the system administrator to have the possibility to send a notification to their dashboard or some area of their Stores that maintenance will be scheduled.

The message could include the date, time, and expected duration. 
It would help everyone to be aware of the maintenance and minimize potential disruptions.

Something like:
"**On December 1st at 12:00 Hrs UTC, there will be a scheduled update, expected to last 2 hours. Take your precautions!**"

**Comment by dennisreimann** [comment]
Good point! @pavlenex @dstrukt

**Reply by pavlenex** [reply]
I think this is good and reasonable feature request. A plugin for server-wide message perhaps?

**Reply by dennisreimann** [reply]
Yeah, this seems to be a good use case for a plugin. Have put that on my list and will let everyone know once there's something installable :)

**Reply by decentralizedb** [reply]
It's amazing to see how well the request has been received.
Thank you 🙏🏽🙏🏽🙏🏽

I will keep an eye on the plugin to test it. 
Congratulations on the project!!!

**Comment by theDavidCoen** [comment]
+1 for a plugin for server-wide general messages.

I have more than 300 users on my BTCpay server running their own stores.
I'd like to have a way to communicate with them by sending notifications (maintenance, issues, new activated plugins...).

**Comment by decentralizedb** [comment]
Hello.
Now that BTCPayserver has added Subscriptions, I think this improvement request is even more relevant.
It would be helpful if operators could notify subscribed users about service outages for maintenance, updates, or other important messages. This way, users can take backups or any other necessary steps in advance.

**Reply by TChukwuleta** [reply]
Hey @decentralizedb  @theDavidCoen 

Implemented an alert notification plugin

https://plugin-builder.btcpayserver.org/public/plugins/serveralert

you can try it out and let us know how it goes and if there be any other feature requests

**Reply by theDavidCoen** [reply]
@TChukwuleta cool, I try asap, thanks!

**Comment by decentralizedb** [comment]
Hello!
I've tried to install it, but it seems it's not compatible with BTCPay Server v2.3.5, because it doesn't show up in the plugin manager, and when I download it and try to install it, it just doesn't appear.It doesn't give any error messages, it just doesn't show up.

I can’t update to BTCPay Server v2.3.6 because it causes issues with the Tickettailor plugin.

Decentralized your life!!!.

Enviado desde un correo electrónico seguro de Proton Mail.

**Reply by TChukwuleta** [reply]
hey @decentralizedb 

2.3.5 have some breaking changes from 2.3.6

Whats the issue with Tickettailor or can you bump Kukks on it.. else I'd have to create a seperate for 2.3.5

**Reply by decentralizedb** [reply]
Hello, thank you for your message.

I've already opened a ticket to report the issue:
https://github.com/Kukks/BTCPayServerPlugins/issues/124

I'll try again as soon as I hear from Kukks about the resolution of the issue.
Thank you very much.

**Comment by pavlenex** [comment]
According to @TChukwuleta this use-case is now possible through https://plugin-builder.btcpayserver.org/public/plugins/serveralert plugin. @decentralizedb @theDavidCoen please give it a try when you guys have a chance and let @TChukwuleta know how it works and if it covers your needs through this plugin.

**Reply by theDavidCoen** [reply]
I confirm it works. Thanks a lot @TChukwuleta !

**Reply by decentralizedb** [reply]
Me too, I confirm it works.
Maybe, as a nice-to-have, a more intrusive notification would be pretty (because it's a maintenance notification), but it works!

Thanks to all for this!

> According to @TChukwuleta this use-case is now possible through https://plugin-builder.btcpayserver.org/public/plugins/serveralert plugin. @decentralizedb @theDavidCoen please give it a try when you guys have a chance and let @TChukwuleta know how it works and if it covers your needs through this plugin.
